### ISSUE TITLE
less cannot parse @font-face's format property

### ISSUE THREAD

**Issue by devgru** [less/less.ruby#48]
Trying to use font-face from CSS3, getting this fail:

```
~$ cat test.less 
@font-face{
font-family: "DejaVu";
src:url(http://devg.ru/fonts/DejaVuLGCSerif.ttf) format("truetype");
}
~$ lessc test.less 
/var/lib/gems/1.8/gems/less-1.0.16/bin/../lib/less/engine/nodes/function.rb:57:in `to_css': undefined method `to_css' for "truetype":String (NoMethodError)
    from /var/lib/gems/1.8/gems/less-1.0.16/bin/../lib/less/engine/nodes/property.rb:115:in `to_css'
    from /var/lib/gems/1.8/gems/less-1.0.16/bin/../lib/less/engine/nodes/property.rb:114:in `map'
    from /var/lib/gems/1.8/gems/less-1.0.16/bin/../lib/less/engine/nodes/property.rb:114:in `to_css'
    from /var/lib/gems/1.8/gems/less-1.0.16/bin/../lib/less/engine/nodes/property.rb:50:in `to_css'
    from /var/lib/gems/1.8/gems/less-1.0.16/bin/../lib/less/engine/nodes/element.rb:133:in `to_css'
    from /var/lib/gems/1.8/gems/less-1.0.16/bin/../lib/less/engine/nodes/element.rb:132:in `map'
    from /var/lib/gems/1.8/gems/less-1.0.16/bin/../lib/less/engine/nodes/element.rb:132:in `to_css'
    from /var/lib/gems/1.8/gems/less-1.0.16/bin/../lib/less/engine/nodes/element.rb:142:in `to_css'
    from /var/lib/gems/1.8/gems/less-1.0.16/bin/../lib/less/engine/nodes/element.rb:141:in `map'
    from /var/lib/gems/1.8/gems/less-1.0.16/bin/../lib/less/engine/nodes/element.rb:141:in `to_css'
    from /var/lib/gems/1.8/gems/less-1.0.16/bin/../lib/less/engine.rb:43:in `to_css'
    from /var/lib/gems/1.8/gems/less-1.0.16/bin/../lib/less/command.rb:56:in `parse'
    from /var/lib/gems/1.8/gems/less-1.0.16/bin/../lib/less/command.rb:49:in `run!'
    from /var/lib/gems/1.8/gems/less-1.0.16/bin/lessc:64
    from /var/lib/gems/1.8/bin/lessc:19:in `load'
    from /var/lib/gems/1.8/bin/lessc:19
~$ 
```

**Comment by cloudhead** [comment]
Should be easy to fix. The `format("truetype")` is the problem here, I still have some work to do on 'functions' in properties, like this one.

**Comment by cloudhead** [comment]
fixed in 1.1.3
